lib/v_0.8.0/templates/client/views/Create.js.jsx

```diff
@@ -40,10 +41,10 @@ class Create__PascalName__ extends Base {
     /**
      * This let's us change arbitrarily nested objects with one pass
      */
-    let new__PascalName__State = _.update( this.state.__camelName__, e.target.name, function() {
+    let newState = _.update( this.state, e.target.name, function() {
       return e.target.value;
     });
-    this.setState({__camelName__ :new__PascalName__State});
+    this.setState({newState});
```

Comment:
remove the curlies from the "newState", else the returned state will have a key called "newState". also change the function() above to an arrow function. same thing in update.